### DISCUSSION TITLE
how to add bootstrap 5 rtl from npm and customize it

### DISCUSSION THREAD

**Discussion by AmirHosseinNP** [twbs/bootstrap#33918] [Q&A] [answered]
Hello, I have downloaded bootstrap 5 from npm and made a custom.scss file. now i want to use rtl and i can't. for example. when i add .ps-5 to an element, it applies the padding to the left. how can i solve this problem?

**Comment by florianlacreuse** [comment]
Hi,

Did you read the doc about RTL? https://getbootstrap.com/docs/5.0/getting-started/rtl/#required-html

**Reply by AmirHosseinNP** [reply]
Hello, yes i did but i couldn't understand what to do! what should i do?! i need to write some code in my custom.scss file to make it happen?! I know i can do it with cdn or the compiled files but then i can't customize the scss variables or maps or other things.

**Reply by florianlacreuse** [reply]
You also need to read this: https://getbootstrap.com/docs/5.0/getting-started/build-tools/#rtlcss

You have to use RTLCSS to process compiled CSS.

**Reply by AmirHosseinNP** [reply]
So you mean I have to use RTLCSS when I wanna do some customization?!

**Reply by AmirHosseinNP** [reply]
I mean I should install RTLCSS from npm or bootstrap have some built it features for doing it?!

**Reply by florianlacreuse** [reply]
I am not aware of a feature in BS to do it, but I never had to deal with RTL so may be I'm wrong.

https://rtlcss.com/learn/usage-guide/install/

**Reply by AmirHosseinNP** [reply]
Oh i got it, anyway thank you for your help🙏

**Reply by ffoodd** [reply]
You need to run RTLCSS on your compiled CSS indeed, as we do in out build. You already run Sass on your side, this is not different.

**Comment by motahharm** [comment, marked as answer]
Hey @AmirHosseinNP 
Hi
You can work with LTR and then change it to RTL using [Rtlcss](https://rtlcss.com/) and use it.

You can use Rtlcss with npm or use online.
[RTLcss Playgorund](https://rtlcss.com/playground/)
